cli/src/plugins/javadoc/index.ts

```diff
@@ -612,110 +625,109 @@ const makeMethodOverloadsDetailBody: MakeBodyFunction<MethodDoc[]> = (args) => {
         }),
-        md.heading(depth + 1, [
-          md.text(doc.modifiers),
-          md.text(" "),
-          project.linkToEntity(
-            doc.returnType.qualifiedTypeName,
-            doc.returnType.typeName
-          ),
-          md.text(` ${doc.name} `),
-          ...makeTypeParameterListWithLinks(project, doc),
-          ...makeParameterListWithLinks(project, doc),
-        ]),
-        tagsToMdast(project, doc.inlineTags),
-
-        // Type parameters section
-        ...makeSection({
-          ...args,
-          depth: depth + 2,
-          title: "Type Parameters",
-          shouldMakeSection: () => doc.typeParamTags.length !== 0,
-          makeBody: () => {
-            return md.list(
-              "unordered",
-              doc.typeParamTags.map((tag) => {
-                return md.listItem([
-                  md.paragraph([
-                    md.inlineCode(`${tag.parameterName}`),
-                    md.text("- "),
-                    tagsToMdast(project, tag.inlineTags ?? []),
-                  ]),
-                ]);
-              })
-            );
-          },
+        project.declareEntity({
+          canonicalName: `${doc.qualifiedName}(${doc.parameters
+            .map((param) => param.type.simpleTypeName)
+            .join(",")})`,
+          pageUri,
         }),
 
-        // Parameters section
-        ...makeSection({
-          ...args,
-          depth: depth + 2,
-          title: "Parameters",
-          shouldMakeSection: () => doc.paramTags.length !== 0,
-          makeBody: () => {
-            return md.list(
-              "unordered",
-              doc.paramTags.map((paramTag) => {
-                return md.listItem([
+        makeMethodDetail(
+          [
+            md.paragraph([
+              md.text(doc.modifiers),
+              md.text(" "),
+              project.linkToEntity(
+                doc.returnType.qualifiedTypeName,
+                doc.returnType.typeName
+              ),
+              md.text(` ${doc.name} `),
+              ...makeTypeParameterListWithLinks(project, doc),
+              ...makeParameterListWithLinks(project, doc),
+            ]),
+          ],
+          [
+            tagsToMdast(project, doc.inlineTags),
+
+            md.paragraph(),
+            // Type Parameters section
+            doc.typeParamTags.length !== 0
+              ? md.paragraph([
+                  md.strong(md.text("Type Parameters")),
+                  md.list(
+                    "unordered",
+                    doc.typeParamTags.map((tag) => {
+                      return md.listItem([
+                        md.paragraph([
+                          md.inlineCode(`${tag.parameterName}`),
+                          md.text("- "),
+                          tagsToMdast(project, tag.inlineTags ?? []),
+                        ]),
+                      ]);
+                    })
+                  ),
+                ])
+              : md.paragraph(),
```

Comment:
Why not use makeSection()?

Reply:
This is inside of a table, and as far as I know you can't make sections inside of tables.

Even if I *could*, I feel like I shouldn't.

Reply:
Oh, I see what's going on now. Thanks.